### DISCUSSION TITLE
部分字符编译后无法显示

### DISCUSSION THREAD

**Discussion by MusicJIA** [stone-zeng/fduthesis#286] [Q&A] [answered]
使用的fduthesis版本为：v0.7e；存在问题：编译后，公式中部分字符无法显示，例如：\alpha、\beta。

**Comment by stone-zeng** [comment]
请提供一下可供编译的代码以及相应的报错信息，此外建议使用最新版（v0.8）。

**Reply by MusicJIA** [reply]
> 请提供一下可供编译的代码以及相应的报错信息，此外建议使用最新版（v0.8）。

代码：

```tex
\begin{equation}
	\mathbf{\beta}_{(t-1)}=softmax(\mathbf{b}_{(t-1)} \odot \mathbf{c}),
\end{equation}
```

编译没有报错信息，可以正常编译。但是，\beta编译后显示为一个小空心方格。
代码中如果将\beta替换为\alpha，也是显示为一个小空心方格。

**Reply by sikouhjw** [reply]
请你告诉我，`\mathbf{\beta}` 是什么意思

**Reply by MusicJIA** [reply]
> 请你告诉我，`\mathbf{\beta}` 是什么意思

将\beta字符加粗显示

**Reply by MusicJIA** [reply]
> 请提供一下可供编译的代码以及相应的报错信息，此外建议使用最新版（v0.8）。

问题已解决，谢谢！

**Comment by note286** [comment, marked as answer]
@MusicJIA 
```latex
\documentclass{fduthesis}
\begin{document}
\mainmatter
\begin{equation}
\symbf{\beta}{(t-1)}=softmax(\symbf{b}{(t-1)} \odot \symbf{c}),
\end{equation}
\backmatter
\end{document}
```

![image](https://user-images.githubusercontent.com/16207292/210687145-578a78fb-7d81-4218-a287-3327cb9ac65b.png)


请使用`\symbf`。

**Reply by MusicJIA** [reply]
> @MusicJIA
> 
> ```
> \documentclass{fduthesis}
> \begin{document}
> \mainmatter
> \begin{equation}
> \symbf{\beta}{(t-1)}=softmax(\symbf{b}{(t-1)} \odot \symbf{c}),
> \end{equation}
> \backmatter
> \end{document}
> ```
> 
> ![image](https://user-images.githubusercontent.com/16207292/210687145-578a78fb-7d81-4218-a287-3327cb9ac65b.png)
> 
> 请使用`\symbf`。

谢谢解答。
